src/support/FabricationDefinition.ts

```diff
@@ -44,68 +44,76 @@ Review Notes:
 Review Notes (Dyoung - 2021.01.11)
 	Changed FileLocator() function to Constructor
 	Moved FileLocator object after File object to maintain alphabetical order of objects.
-	Added RUN and INCLUDE to SPECIALTYPES KEywork Group
+	Added RUN and INCLUDE to SPECIALTYPES Keyword Group
 	Minor updates/misspelling in documentation in various locations
-	
+Review Notes (DYoung = 2021.01.25)
+	Added 'BREAKPOINT' to SPEACIALTYPES
+	Added 'OUTPUT()' to Global Functions
+	Edited Description for 'ERROR' Global Function
+	Added 11 'Add____()' and 11 'Remove____()' Methods to 'PRODUCTENTRY' Object.
+	Added 'SEEKENTY()' and 'SEEKLINE()' Methods to 'FILE' Object.
+	Added 'ADDLINK()', 'DELETELINK()' and 'SETDONOTCUTFLAG()' Methods to 'ITEMSTRUCT' Object.
 */
 
 // IMPORTANT: Anywhere (in code) you are representing an actual value of a string, please use single quotes. This does not apply to the /** documentation areas */
 
 
 // This namespace represents a template of the entire Autodesk Fabrication scripting environment
 // There are some caveats to this for OPERATORS, NUMBERS & STRINGS
-export namespace FABRICATION {
+namespace FABRICATION {
 
 	//#region Fabrication Keyword Groups
 
-	export const FLOWCONTROL: Array<string> = [
-		'If',
-		'Then', 
-		'Else', 
-		'Else If', 
-		'ElseIf', 
-		'End If', 
-		'EndIf', 
-		'While', 
-		'End While', 
-		'EndWhile', 
-		'For', 
-		'To', 
-		'Step', 
-		'Next', 
-		'Do', 
-		'Until', 
-		'Loop', 
-		'Select', 
-		'Case', 
-		'End Select', 
-		'EndSelect', 
-		'Function', 
-		'Return', 
-		'End Function', 
-		'EndFunction',		
+	const FLOWCONTROL: Array<string> = [
+		'IF',
+		'THEN', 
+		'ELSE', 
+		'ELSE IF', 
+		'ELSEIF', 
+		'END IF', 
+		'ENDIF', 
+		'WHILE', 
+		'END WHILE', 
+		'ENDWHILE', 
+		'FOR', 
+		'TO', 
+		'STEP', 
+		'NEXT', 
+		'DO', 
+		'UNTIL', 
+		'LOOP', 
+		'LOOPUNTIL', 
+		'SELECT', 
+		'CASE', 
+		'END SELECT', 
+		'ENDSELECT', 
+		'FUNCTION', 
+		'RETURN', 
+		'END FUNCTION', 
+		'ENDFUNCTION',		
 	];
 	
 	// Note: The entire enumeration section further below is merged into this list for highlighting purposes when generating codscript-tmLanguage.json
-	export const VALUETYPES: Array<string> = [
-		'True',
-		'False', 
-		'Null', 
-		'Void'
+	const VALUETYPES: Array<string> = [
+		'TRUE',
+		'FALSE', 
+		'NULL', 
+		'VOID'
 	];
 
-	export const SPECIALTYPES: Array<string> = [
-		'And', 
-		'Or', 
-		'Not', 
-		'As', 
-		'Is', 
-		'New', 
-		'Dim', 
-		'Object', 
-		'Requires',
-		'Include',
-		'Run'
+	const SPECIALTYPES: Array<string> = [
+		'AND', 
+		'OR', 
+		'NOT', 
+		'AS', 
+		'IS', 
+		'NEW', 
+		'DIM', 
+		'OBJECT', 
+		'REQUIRES',
+		'INCLUDE',
+		'RUN',
+		'BREAKPOINT'
```

Comment:
make all of these in green resemble the ones in red, they were adjusted to be "pretty" since they are directly affecting the suggestions made during auto-completion.

Also notice there is an `export` keyword on the front of them that we do in fact need.

Reply:
Note to self: completed locally. Resolve after update committed.